neptune/new/runs_table.py

```diff
@@ -17,44 +17,20 @@
 # backwards compatibility
 # pylint: disable=unused-import,wrong-import-order
 from neptune.new.internal.backends.api_model import AttributeType
-from neptune.new.attributes_containers_table import LeaderboardEntry, LeaderboardHandler
+from neptune.new.metadata_containers.metadata_containers_table import (
+    LeaderboardEntry,
+    LeaderboardHandler,
+)
 from neptune.new.exceptions import MetadataInconsistency
 
-#
-
-import logging
-from typing import List
-
-from neptune.new.attributes_containers_table import (
-    AttributesContainersTable,
-    AttributesContainersTableEntry,
+from neptune.new.metadata_containers.metadata_containers_table import (
+    MetadataContainersTable,
+    MetadataContainersTableEntry,
 )
 from neptune.new.internal.backends.api_model import (
     AttributeWithProperties,
 )
 from neptune.new.internal.backends.neptune_backend import NeptuneBackend
 from neptune.new.internal.container_type import ContainerType
 
-logger = logging.getLogger(__name__)
-
-
-class RunsTableEntry(AttributesContainersTableEntry):
-    def __init__(
-        self,
-        backend: NeptuneBackend,
-        _id: str,
-        attributes: List[AttributeWithProperties],
-    ):
-        super().__init__(
-            backend=backend,
-            container_type=ContainerType.RUN,
-            _id=_id,
-            attributes=attributes,
-        )
-
-
-class RunsTable(AttributesContainersTable):
-    table_entry_cls = RunsTableEntry
-
-    def to_runs(self):
-        return self.to_table_entries()
+from neptune.new.metadata_containers.runs_table import RunsTable, RunsTableEntry
```

Comment:
I know you have a separate issue for deprecation https://neptune-labs.atlassian.net/browse/NPT-11395 , but please either leave an easily greppable comment in files like that or add them to NPT-11395 description

Reply:
Isn't

```
# backwards compatibility
# pylint: disable=unused-import,wrong-import-order
```

enough to find such places?

Reply:
right, sorry -- I've seen you've removed from https://github.com/neptune-ai/neptune-client/pull/811/files#diff-51310aa42e9441fc7625038b9dffa9f353864634268799198b624f273f74f4ecL29 and got lost a little bit